lib/src/utils/validators.dart

```diff
@@ -0,0 +1,41 @@
+class Validators {
+  const Validators._();
+
+  static String? cardNumberValidator(String? value, String errorMsg) {
+    // Validate less that 13 digits +3 white spaces
+    return (value?.isEmpty ?? true) || (value?.length ?? 16) < 16
+        ? errorMsg
+        : null;
+  }
+
+  static String? expiryDateValidator(String? value, String errorMsg) {
+    if (value?.isEmpty ?? true) {
+      return errorMsg;
+    }
+
+    final DateTime now = DateTime.now();
+    final List<String> date = value!.split(RegExp(r'/'));
+
+    final int month = int.parse(date.first);
+    final int year = int.parse('20${date.last}');
```

Comment:
why are we using 20 as constant here? `int.parse('20${date.last}');`

Reply:
I have just extracted already written code to this class. But, this code basically validates the expiry date which is in format MM/YY. Thus this logic would be valid till the year 2099. 
We can switch to using the first two digits dynamically as per the current year, but both the approaches would fail for edge cases like when current year is 2099 and expiry year is 2100.